src/views/portals/utils/DashboardImagesGallery.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="box">
+    <h4 class="title is-3">
+      <router-link to="images">
+        <icon-image />
+        {{ $gettext('images') | uppercaseFirstLetter }}
+      </router-link>
+    </h4>
+    <div class="images-container">
+      <loading-notification :promise="imagesPromise" />
+      <gallery v-if="images != null" :images="images.documents" />
+    </div>
+    <hr />
+    <h6 class="title is-6 has-text-centered">
+      <router-link to="images">
+        <span v-translate>See more</span>
+      </router-link>
+    </h6>
+  </div>
+</template>
+
+<script>
+import Gallery from '@/components/gallery/Gallery';
+import c2c from '@/js/apis/c2c';
+
+export default {
+  name: 'DashboardImagesGallery',
+
+  components: {
+    Gallery,
+  },
+
+  data() {
+    return {
+      imagesPromise: null,
+    };
+  },
+
+  computed: {
+    images() {
+      return this.imagesPromise?.data;
+    },
+  },
+
+  created() {
+    this.imagesPromise = c2c.image.getAll();
```

Comment:
this, by default, retrieves 30 docs. The image gallery is a small box that will show at most 2 images at the same time. I am pretty sure 99,99 of users will never slide, but use the `See more` link instead => this should be reworked IMHO.

Reply:
Limited to 10 images seems to be ok for me